xikolo-ios/ViewControllers/Courses/CertificatesTableViewController.swift

```diff
@@ -0,0 +1,88 @@
+//
+//  Created for xikolo-ios under MIT license.
+//  Copyright © HPI. All rights reserved.
+//
+
+import UIKit
+import SafariServices
+
+class CertificatesTableViewController: UITableViewController {
+
+    var course: Course!
+
+    // MARK: - Table view data source
+
+    override func numberOfSections(in tableView: UITableView) -> Int {
+        return 1
+    }
+
+    override func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return section == 0 ? 3 : 0
+    }
+
+    override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        let cell = tableView.dequeueReusableCell(withIdentifier: "certificateCell", for: indexPath)
+        switch indexPath.row {
+        case 0:
```

Comment:
To avoid to switch statements, you can create a helper method that returns the title and url optional for a given index path. Then you can run the code of `certificateState(_ certificateURL: URL?)` in this method. This makes it easier to reorder the certificates.